.github/workflows/issue-tests.yml

```diff
@@ -47,5 +47,4 @@ jobs:
     - name: Test Issues Against YB Version
       run: |
         cd yb-voyager
-        go test -v -run '^(TestDDLIssuesInYBVersion|TestDMLIssuesInYBVersion)$' ./...
-        
+        go test -v -run '^(TestDDLIssuesInYBVersion|TestDMLIssuesInYBVersion)$' ./... -tags 'integration'
```

Comment:
we should skip `integration` here, right?

Reply:
yes, will change this.